frontend/src/app/actions/newsletter-form.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from 'redux-actions';
+
+const actions = createActions(
+  {
+    newsletterFormSetEmail: email => email,
+    newsletterFormSetPrivacy: privacy => privacy
+  },
+  'newsletterFormSetFailed',
+  'newsletterFormSetSubmitting',
+  'newsletterFormSetSucceeded'
+);
+
+const subscribeActions = createActions({
+  newsletterFormSubscribe: (dispatch, email) => {
```

Comment:
`dispatch` is added in `mapDispatchToProps` here: https://github.com/mozilla/testpilot/pull/1620/files#diff-d1e3fb4bae03a0b6b298c56e7110d06fR159